### DISCUSSION TITLE
Does this work for v.0.4 koms (els)?

### DISCUSSION THREAD

**Discussion by kelvinp12** [Elskom/Els_kom_new#191] [Q&A] [answered]
Wondering since it did get updates, but making sure as the procedures seems quite tricky..
And if it does, how about the textures (.kuri)? Do they get decrypted as well?

Only looking for extractor, if it works. Better if it works with the steam release (not JP nor EU ver)

**Comment by AraHaan** [comment, marked as answer]
Currently all of the extracting can be done by plugins since the files are encrypted and so the keys always change and so I pushed it to where they are managed by plugins so those who make the plugins can worry about that as an implementation detail on their end.

As for v4 support I had one on the works but it kept crashing the program and so I made all of the kom plugins private for now until I figure out the issue on unpacking the header xml again.

**Reply by kelvinp12** [reply]
So v4 doesnt work yet, I take it?

Extracting by plugins? Is that any different to the old kom unpackers/repackers? 
(Don't need the latter, but I'm rather confused of what you meant)

**Reply by AraHaan** [reply]
It is different yes because they do not implement a plugin system so they can break at any time and go unmaintained.

The plugin system is better because then you can update them.

Note: I might make some changes to the plugin system to require plugins to be installed as nuget packages and with configurable nuget feeds to install them.

**Reply by animation31** [reply]
Hello!
Hope you're doing well! I would like to humbly ask if the v4 is still in the works? and does it work with Grand Chase?
